.gitignore

```diff
@@ -9,3 +9,7 @@ integration/.idea/
 integration/.tox/
 integration/.venv/
 integration/MANIFEST
+cscope.files
+cscope.out
+f1
+f2
```

Comment:
is it required ?

Reply:
yes, just to ignore the files used while integration testing and for other purpose